packages/sdk/src/types/Products.ts

```diff
@@ -7,14 +13,6 @@ export interface ProductsCreate {
 	image_url?: string
 }
 
-export interface ProductsGetResponseSkus {
```

Comment:
It looks like `ProductsGetResponseSkus` was removed. We have to put it back or describe it in the changelog.